alexthompson207/current-climate
# Feature/all articles view

# PR CurrentClimate

### What does this do?


### Are there any known bugs?


### Summary of the changes
Create a component directory, add App, Story, Header, NewsView component folders and respective JS and CSS files. 
Add basic structure to Story and NewsView component and basic styling.

### What ticket(s) do the changes resolve?
#6 closes

Please Link Issues to this PR as necessary.

### Notes for the Reviewer
N/A

### How to Test Changes?
Clone down repo, open site locally, should see 2 stories with the same content being displayed on the local site.

### Next Iterations
- Create a Header component and add basic styling
- GET data from API and connect with application
